internal/service/contentprovider/moduleconfig.go

```diff
@@ -69,17 +69,52 @@ func (s *ModuleConfigProvider) validateArgs(args types.KeyValueArgs) error {
 }
 
 type ModuleConfig struct {
-	Name          string            `yaml:"name" comment:"required, the name of the Module"`
-	Version       string            `yaml:"version" comment:"required, the version of the Module"`
-	Channel       string            `yaml:"channel" comment:"required, channel that should be used in the ModuleTemplate"`
-	ManifestPath  string            `yaml:"manifest" comment:"required, relative path or remote URL to the manifests"`
-	Mandatory     bool              `yaml:"mandatory" comment:"optional, default=false, indicates whether the module is mandatory to be installed on all clusters"`
-	DefaultCRPath string            `yaml:"defaultCR" comment:"optional, relative path or remote URL to a YAML file containing the default CR for the module"`
-	ResourceName  string            `yaml:"resourceName" comment:"optional, default={name}-{channel}, when channel is 'none', the default is {name}-{version}, the name for the ModuleTemplate that will be created"`
-	Namespace     string            `yaml:"namespace" comment:"optional, default=kcp-system, the namespace where the ModuleTemplate will be deployed"`
-	Security      string            `yaml:"security" comment:"optional, name of the security scanners config file"`
-	Internal      bool              `yaml:"internal" comment:"optional, default=false, determines whether the ModuleTemplate should have the internal flag or not"`
-	Beta          bool              `yaml:"beta" comment:"optional, default=false, determines whether the ModuleTemplate should have the beta flag or not"`
-	Labels        map[string]string `yaml:"labels" comment:"optional, additional labels for the ModuleTemplate"`
-	Annotations   map[string]string `yaml:"annotations" comment:"optional, additional annotations for the ModuleTemplate"`
+	Name             string            `yaml:"name" comment:"required, the name of the Module"`
+	Version          string            `yaml:"version" comment:"required, the version of the Module"`
+	Channel          string            `yaml:"channel" comment:"required, channel that should be used in the ModuleTemplate"`
+	Manifest         string            `yaml:"manifest" comment:"required, relative path or remote URL to the manifests"`
+	ManifestFilePath string            `yaml:"-"` // ignore this field, will be filled programmatically
```

Comment:
Need to think about this one more detailed.

Reply:
Okay, think I know what to do. But may be a medium sized refactoring.

Reply:
Proposal for fixing this is here: https://github.com/c-pius/modulectl/pull/1

We can't (or shouldn't) simply re-apply `GetManifestPath` because this will re-download the file every time and put it into a new temp file.